src/App.css

```diff
@@ -2,4 +2,20 @@
 
 * {
     font-family: 'Montserrat', sans-serif;
+}
```

Comment:
Fyi: you could use styled-component's [createGlobalStyle](https://styled-components.com/docs/api#createglobalstyle) instead of a `.css` file. Just to be more consistent. It's up to you though if you want to keep it this way or use `createGlobalStyle`